src/test/kotlin/no/nav/familie/ef/sak/beregning/BeregningUtilsTest.kt

```diff
@@ -92,11 +92,11 @@ internal class BeregningUtilsTest {
             val inntektsperioder: List<Inntektsperiode> =
                     listOf(Inntektsperiode(LocalDate.of(2021, 1, 1),
                                            LocalDate.of(2021, 4, 30),
-                                           BigDecimal(1000),
+                                           200000.toBigDecimal(),
```

Comment:
Kanskje ikke veldig viktig i testene, men for økt lesbarhet så kan man legge inn tusenseparator _
```suggestion
                                           200_000.toBigDecimal(),
```